etc/nginx/snippets/rewrites.conf

```diff
@@ -4,9 +4,9 @@ set $current_version_sdk_c_api '3.3.0';
 set $current_version_sdk_dotnet_api '3.3.1';
 set $current_version_sdk_go_api '2.5.0';
 set $current_version_sdk_java_api '3.3.1';
-set $current_version_sdk_jvm_core_api '2.3.0';
+set $current_version_sdk_jvm_core_api '2.3.1';
```

Comment:
unrelated to node.js 4.1.1?

Reply:
Someone forgot to put it in with the Java release - just a drive-by fix  ;)